bedrock/base/templates/404.html

```diff
@@ -2,47 +2,63 @@
-
 {% block page_css %}
   {{ css_bundle('page_not_found') }}
 {% endblock %}
 
+{# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
+{% block canonical_urls %}{% endblock %}
+
 {% block content %}
-<main>
-  <div class="content">
-    <header>
-      <h1><span>{{ _('Whoops!') }}</span></h1>
-      <p>{{ _('Did you make a left at that last URL instead of a right? No problem. Here are some tips to get you back on your way:') }}</p>
-    </header>
-    <div class="error-content">
-      <ul class="prose">
-        <li>{{ _('If you typed in the address, check your spelling. Could just be a typo.') }}</li>
+  <main role="main">
+    <section class="mzp-l-content message-section">
+      <h2>{{ _("Sorry, we can't find that page") }}</h2>
+      <p>{{_("We're all about a healthy internet but sometimes broken URLs happen.") }}</p>
+      <div>
+        <a class="back-button" href="javascript:history.back()">{{ _("Go Back") }}</a>
+      </div>
+    </section>
+    <section class="mzp-l-content">
+      <ul>
         <li>
-        {% trans bugzilla='https://bugzilla.mozilla.org/enter_bug.cgi?product=www.mozilla.org' %}
-          If you’ve found an issue with one of our websites, we’d appreciate it if you could report the problem in Bugzilla, our <a href="{{ bugzilla }}">bug tracker</a>. One of our developers will take a look at it as soon as possible.
-        {% endtrans %}
+          <section class="mzp-has-icon">
+            <img class="list-icon" alt="" src="{{ static('protocol/img/icons/mozilla.svg') }}" width="30" height="30">
+              <p class="list-content">
+                {% trans about=url('mozorg.about') %}
+                  <a class="no-highlight" href="{{ about }}">Learn</a> about Mozilla, the non-profit behind Firefox.
+                {% endtrans %}
+              </p>
+            
+          </section>
         </li>
-        <li>{{ _('If you followed a link, it’s probably broken.') }}</li>
         <li>
-        {% trans  url=url('mozorg.home') %}
-          If you’re not sure what you’re looking for, start at <a href="{{ url }}">mozilla.org</a>.
-        {% endtrans %}
+          <section class="mzp-has-icon">
+            <img class="list-icon" alt="" src="{{ static('img/base-error/firefox_logomark_black.svg') }}" width="30" height="30">
+            <p class="list-content">
+            {% trans explore=url('firefox') %}
+              <a class="no-highlight" href={{ explore }}>Explore</a> the entire family for Firefox products designed to respect your privacy.
+            {% endtrans %}
+            </p>
+          </section>
         </li>
+        <li>
+          <section class="mzp-has-icon">
+            <img class="list-icon" alt="" src="{{ static('protocol/img/icons/desktop.svg') }}" width="30" height="30" color="black">
+            <p class="list-content">
+            {% trans  download='https://firefox.com/new'%}
```

Comment:
Currently this link redirects to /new, so until firefox.com exists as a real project I'd suggest linking directly?
```suggestion
            {% trans download=url('firefox.new') %}
```